blues/sparrow-starter
# fix: sensor name and location were not shown initially in the details tab

# Problem Context

When the Details tab is displayed, the sensor name and location are empty, even if the sensor has these values defined. 

## Changes

The antd form component is wrapped, for simplicity I'm guessing. Each form item is created from a `FormProps` defintion.  This PR adds `initialValue` to `FormProps` so the initial value can be specified. 

This was originally based on `main` but PR #47 also adds the location property to Sensor with all the plumbing, so I rebased on that.


## Screenshot (if applicable)


## Testing

Unit / integration / e2e tests?
Steps to test manually?

* view the sensor details page, the name and location should reflect the currently set name and location. 

Any other sorts of testing notes to include?

This page will likely go under significant rework so e2e tests are not critical at the present time.

## Any other related PRs

## Ticket(s)

Addresses the "sensor name and location are shown" requirement in ticket
https://www.pivotaltracker.com/story/show/181159368.
